README.md

```diff
@@ -1 +1,119 @@
-# steampipe-plugin-doppler
\ No newline at end of file
+![image](https://hub.steampipe.io/images/plugins/turbot/doppler-social-graphic.png)
+
+# Doppler Plugin for Steampipe
+
+Use SQL to query projects, environments, secrets and more from Doppler.
+
+- **[Get started →](https://hub.steampipe.io/plugins/turbot/doppler)**
+- Documentation: [Table definitions & examples](https://hub.steampipe.io/plugins/turbot/doppler/tables)
+- Community: [Slack Channel](https://steampipe.io/community/join)
+- Get involved: [Issues](https://github.com/turbot/steampipe-plugin-doppler/issues)
+
+## Quick start
+
+Download and install the latest Doppler plugin:
+
+```bash
+steampipe plugin install doppler
+```
+
+Configure your [credentials](https://hub.steampipe.io/plugins/turbot/doppler#credentials) and [config file](https://hub.steampipe.io/plugins/turbot/doppler#configuration).
+
+### Configuring Doppler Credentials
```

Comment:
```suggestion
```

Reply:
Do we need to remove this one?

Reply:
Kindly validate as the namecheap plugin, we follow the same standard.